app/src/main/java/com/rentminder/SaveButton.kt

```diff
@@ -35,33 +21,38 @@ import androidx.compose.ui.unit.dp
 fun SaveRemindButton(enabled: Boolean, inputEdited: MutableState<Boolean>) {
     val context = LocalContext.current
     val focusManager = LocalFocusManager.current
-    var count by remember { mutableStateOf(0) }
+    var saved by remember { mutableStateOf(false) }
```

Comment:
Switched from count and mutableStateOf 0 to a boolean in the state of true or false to simply see if a save has occurred or not.